src/systemcmds/tests/test_dataman.c

```diff
@@ -63,6 +63,16 @@ int test_dataman(int argc, char *argv[]);
 
 #define NUM_MISSIONS_TEST 50
 
+typedef union dataman_max_size_t {
+	struct home_position_s		safe_item;
+	struct mission_item_s		mission_item;
+	struct mission_s			mission;
+	struct fence_vertex_s		vertex;
+	struct dataman_compat_s		compat;
+} dataman_max_size_t;
+
+#define DM_MAX_DATA_SIZE sizeof(dataman_max_size_t)
```

Comment:
Well I just moved it I think is good keep this way in case the structs change size and other struct becomes bigger than struct mission_s also the compiler will optimize it anyway.